### ISSUE TITLE
Deployment not erroring if syntax errors in Calculation Item

### ISSUE THREAD

**Issue by o-o00o-o** [otykier/TabularEditor#378]
Hi Daniel, we had a late found issue in our calculation items that I would have expected any deployment would have errored with.

So we have this calculation item that has a syntax error in

![image](https://user-images.githubusercontent.com/13693159/71817565-20a63f00-307e-11ea-9d80-2ff046256d2c.png)

AAS is reporting the error correctly however I would have expected that when we deployed this we should have got a fatal exception

Our Command line is built from this PS function:

``` powershell
 function New-TabEditorArgList {
        param($CubeFileName, $Instance, $Catalog, $Login = '', $DatabaseInstance, $DatabaseCatalog)

        return "$CubeFileName -DEPLOY $Instance $Catalog $Login -OVERWRITE -CONNECTIONS %server% $DatabaseInstance %database% $DatabaseCatalog -PARTITIONS -ROLES -MEMBERS -VSTS -ERR"
    }
```

but we got this output

```
2020-01-06T00:33:12.2282840Z VERBOSE: Performing the operation "Deploy Cube" on target 
2020-01-06T00:33:12.2283942Z "C:\azagent\A1\_work\r2\a\_VegaDW-CI\Deployment Package\Model.bim".
2020-01-06T00:33:49.9558594Z 
2020-01-06T00:33:49.9566287Z Tabular Editor 2.9.1 (build 2.9.7282.17172)
2020-01-06T00:33:49.9573055Z --------------------------------
2020-01-06T00:33:49.9580411Z Dependency tree built in 532 ms
2020-01-06T00:33:49.9588174Z Loading model...
2020-01-06T00:33:49.9595335Z Switching connection string placeholders...
2020-01-06T00:33:49.9602299Z Deploying...
2020-01-06T00:33:49.9609321Z Deployment succeeded.
2020-01-06T00:33:49.9615421Z Information: Unprocessed object: partition 'Emails 2015' on table 'Emails' (NoData)
2020-01-06T00:33:49.9623396Z Information: Unprocessed object: partition 'Emails 2016' on table 'Emails' (NoData)
2020-01-06T00:33:49.9630756Z Information: Unprocessed object: partition 'Emails 2017' on table 'Emails' (NoData)
2020-01-06T00:33:49.9639029Z Information: Unprocessed object: partition 'Emails 2018' on table 'Emails' (NoData)
2020-01-06T00:33:49.9703626Z VERBOSE: Replacing datasource
2020-01-06T00:33:58.8675318Z <return xmlns="urn:schemas-microsoft-com:xml-analysis"><root xmlns="urn:schemas-microsoft-com:xml-analysis:empty"></root></return>
2020-01-06T00:33:59.4018904Z VERBOSE: DONE:  Deploying Tabular Cube. Duration: 00:00:47.2317545
```

Pretty sure if I put this type of issue in a measure it would fail the deploy. Is Tabular Editor not checking for model errors on calculation items?

**Comment by otykier** [comment]
Just to clarify: Tabular Editor will never fail/rollback a deployment in case of errors - but certain structural errors may cause Analysis Services to reject the deployment. In case of errors on measures, calculated tables, row-level filters, etc., the deployment will still happen, but Tabular Editor reports the error back to the console, so that you can react accordingly in your build pipeline.

It seems 2.9.1 has a bug in which errors on calculation items are not reported to the console (even though they are shown in the Tabular Editor UI). I will look into that.

**Comment by otykier** [comment]
Fixed in [2.9.3](https://github.com/otykier/TabularEditor/releases/tag/2.9.3)
